ci-operator/step-registry/telco5g/ptp-cluster-setup/telco5g-ptp-cluster-setup-commands.sh

```
@@ -306,10 +306,29 @@ if [[ "$SKIP_OCP_INSTALL" != "true" ]]; then
   ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/ocp-install.yml -e job_type=$JOB_TYPE -vv || status=$?
 fi
 
-# Fetch kubeconfig and cluster information
-ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -e job_type=$JOB_TYPE -vv || true
+# Fetch kubeconfig and cluster information. Do not ignore failures: without kubeconfig the test
+# step has nothing to work with, and the job can spuriously pass the setup ref while the test
+# step then fails to upload artifacts (e.g. e2e-telco5g-ptp-upstream with SKIP_OCP_INSTALL=true).
+if ! ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -e job_type=$JOB_TYPE -vv; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)